### PR TITLE
Add CamemBERT models to modeling_auto

### PR DESCRIPTION
**Pull request by Evpok**
CamemBERT was in autoconfig but not in automodel, this PR aims to correct that. Also first PR here so please tell me if I missed some things :-)

### CODE REVIEW

File: transformers/modeling_auto.py [hunk 1]
```diff
@@ -172,6 +177,7 @@ class method.
         The model class to instantiate is selected as the first pattern matching
         in the `pretrained_model_name_or_path` string (in the following order):
             - contains `distilbert`: DistilBertForMaskedLM (DistilBERT model)
+            - contains `camembert`: CamemBERTModel (CamemBERT model)
```
Comment (thomwolf) [inline transformers/modeling_auto.py]: This `CamemBERTModel` should be `CamembertForMaskedLM`

Comment (Evpok) [reply transformers/modeling_auto.py]: Indeed, too many copypasta, thanks

**Review by thomwolf — CHANGES_REQUESTED** [review]
Thanks for that. A few things to fix and then @LysandreJik can have a look as well.

**Comment by LysandreJik** [comment]
Great, thanks @Evpok !

**Pull request merged**